src/Symfony/Bundle/FrameworkBundle/Resources/config/mailer_transports.xml

```diff
@@ -4,6 +4,10 @@
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
+    <parameters>
+        <parameter key="mailer.sendmail_default_command">/usr/sbin/sendmail -bs</parameter>
```

Comment:
We try to not add parameters. I would go with a semantic config instead.

Reply:
I see some drawbacks to use a configuration:
- relation between a config and a transport must be hardcoded
- if we don't want to hardcode the relation between a config and a transport, we can use a generic array (i.e. `options`) but in this case we can't validate config parameters at compile time (for instance to alert that a key used in `options` isn't used by any transport)
- not sure there is many other cases where an internal configuration setting (not related to the dsn) would be useful.

An other approach to solve the original problem (use the sendmail path and the sendmail options that work in our environment) could be to use a `NativeTransportFactory` which use sendmail command or smtp server configured in php.ini. This new try is an implementation of this approach.